doc/manuals/admin/diagnosis.md

```diff
@@ -258,7 +258,7 @@ Some possible values for `lastFailureReason` (non exahustive list):
 
 More detail about `status`, `lastFailureReason` and `lastSuccessCode` in the
```

Comment:
Replace the reference to NGSIv2 spec by the reference to Orion API

No direct reference to NGSIv2 spec should remain in the documentation. The reference in the Orion API preface should suffice. I'll add this as an item to #4141

Reply:
Done in caaddeea98a0253ca07fad4cc04036d04b1fc3a0